src/ert/_c_wrappers/job_queue/qstat_proxy.sh

```diff
@@ -84,30 +88,22 @@ fi
 # The file is potentially updated:
 proxyage_seconds=`file_age_seconds $proxyfile`
 
-if [ $proxyage_seconds -gt $CACHE_INVALID ]; then
-    echo "qstat_proxy: proxyfile $proxyfile was too old ($proxyage_seconds seconds). Giving up!"
+fail() {
+    echo "$1"
     exit 1
-fi
+}
 
-# Replicate qstat's error behaviour:
-if [ -n "$1" ]; then
-    grep -e "Job Id: ${1}" $proxyfile >/dev/null 2>&1 || {
-        echo "qstat: Unknown Job Id $1" && cat $proxyfile >&2 && exit 1;
-        }
+if [ $proxyage_seconds -gt $CACHE_INVALID ]; then
+    fail "qstat_proxy: proxyfile $proxyfile was too old ($proxyage_seconds seconds). Giving up!"
 fi
 
-# Extract the job id from the proxyfile:
 if [ -n "$1" ]; then
-    awk "BEGIN { RS=\"Job Id: \"} /^$1/,/Job/ {printf \"Job Id: \"; print}" $proxyfile \
-        | grep -v ^$
-    exit 0
+    grep -A $MAX_JOB_LINES "Job Id: $1" $proxyfile || fail "qstat: Unknown Job Id $1"
```

Comment:
Noe that this grep filter is quite simple, if you as for job id say `100` it would also give you job id `1000`. But that should be acceptable, ERT makes sure it fetches the correct job id.

Reply:
` "Job Id: $1$" ` where the last `$1` marks end of line - would it work?

Reply:
That means we require any namespace to always be included, `15449.s034-lcam` is a typical job id with namespace, but `qstat 15449` also works for the backend qstat. That means filtering on end-of-line is risky and I prefer not to do it. ERTs torque driver handles the namespace issue.